### ISSUE TITLE
Error: Unable to read Docker image into resource: unable to list Docker images: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.41/images/json"

### ISSUE THREAD

**Issue by zelden0** [coder/coder#11627]
Hi,
I'm using the deb version of coder

I successfully created only one docker with coder, and after this one, Terraform get stuck (~3 min after the start of the process).
Outside of coder, Docker works fine.
When I stop it, the error is:
`Error: Unable to read Docker image into resource: unable to list Docker images: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.41/images/json": context canceled
on main.tf line 158, in resource "docker_image" "main":`
`   158: resource "docker_image" "main" {`

**Comment by matifali** [comment]
Are you using the included docker starter template?

**Comment by zelden0** [comment]
Yes

**Comment by zelden0** [comment]
it wasn't but that doesn’t change anything
`cat /etc/group`
`docker:x:997:user,netdata,coder,overgeek`

**Comment by matifali** [comment]
Please see the docs [here](https://coder.com/docs/v2/latest/install/docker#permission-denied-while-trying-to-connect-to-the-docker-daemon-socket)

**Comment by tmfksoft** [comment]
I've had the same error, after manually running the templates Dockerfile I found it was hanging because of timezone configuration:
![image](https://github.com/coder/coder/assets/4118296/a6c6d7da-ff1a-4adb-9af4-a0985ce0c3c2)

Versions:
OS: Ubuntu 23.10
Docker Engine: 26.0.0 (API 1.45)
Coder Server: v2.9.1+2a2fd70

I installed via the install script provided and ran the provided Docker template.

I upgraded Ubuntu from 20.04 and tried each step without luck until I edited the Docker template to add
`ARG DEBIAN_FRONTEND=noninteractive`

This fixed it.

Is there a better solution?
